aten/src/ATen/native/Convolution.cpp

```diff
@@ -818,7 +818,7 @@ static auto view3d(const at::Tensor& tensor) -> at::Tensor {
   return tensor.squeeze(2);
 }
 
-static at::Tensor subtensor(at::Tensor& tensor, int dim, int groups, int g) {
+static at::Tensor subtensor(at::Tensor& tensor, int64_t dim, int64_t groups, int64_t g) {
```

Comment:
We support int64 sized groups? Our dim count is hardcoded way lower than int32max too

Reply:
The same reason, the call sites pass int64